server/src/main/java/org/elasticsearch/cluster/metadata/DataStream.java

```diff
@@ -265,7 +260,7 @@ public void insertTimestampFieldMapping(Map<String, Object> mappings) {
                 current = map;
             }
             current.put(leafFieldName, fieldMapping);
-            XContentHelper.update(mappings, changes, false);
+            return changes;
```

Comment:
It's a bit strange to call this "changes" since we're retrieving the mapping for a single field in this method?

Reply:
👍